docs/content/installation/install_hostdefinition.md

```diff
@@ -0,0 +1,36 @@
+# Installing the `HostDefinition` custom resource
+
+Install the `HostDefinition` custom resource to enable automatic host definitions from within the CSI driver.
+
+Use the following steps to install the `HostDefinition` custom resource, with [GitHub](https://github.com/IBM/ibm-block-csi-operator).
+
+**Note:** Only hosts added after installation are automatically defined with the CSI driver.
+
+1.  Install the operator.
+
+    1. Download the manifest from GitHub.
+
+        ```
+        curl https://raw.githubusercontent.com/IBM/ibm-block-csi-operator/v1.10.0/deploy/installer/generated/host-definition.yaml > host-definition.yaml
+        ```
+
+    2. Verify that the hostdefinition is in the _Ready_ phase.
+
+        ```
+        $> kubectl get hostdefinition
+        NAME                  AGE    PHASE   STORAGE          HOST
+        <k8s_resource_name1>  102m   Ready   <storage_name>   <host_name_w1>
+        <k8s_resource_name2>  102m   Ready   <storage_name>   <host_name_w2>
```

Comment:
is `storage_name` really the name of the storage system? or just its management address?

Reply:
the `management address`

Reply:
then let's use a placeholder accordingly

Reply:
you are talking about the `Storage` field?

Reply:
the example placeholders of it, yes

Reply:
<storage_address> ?
<storage_management_address>?

Reply:
updated

Reply:
I think <[management_address](https://github.com/IBM/ibm-block-csi-driver/blob/v1.9.0/deploy/kubernetes/examples/demo-secret.yaml#L8)> is most accurate

Reply:
updated